.github/scripts/tokens/create_issue.sh

```
@@ -0,0 +1,22 @@
+const npmValid = "${{ steps.check_npm.outputs.status }}" === "valid";
```

Shellcheck (0.11.0)

[error] Parameter expansions can't start with {. Double check syntax.

(SC2296)